.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -303,6 +306,28 @@ jobs:
         done
         gsutil cp -r gs://definitions.stackrox.io/v4/vulnerability-bundles/dev/* gs://definitions.stackrox.io/v4/vulnerability-bundles/1.0.0/
 
+    - name: Copy v1 to pre-versioned bundles
+      run: |
+        # using v1 bundle for released versions listed in RELEASE_VERSION
+        single=gs://definitions.stackrox.io/v4/vulnerability-bundles/v1/vulns.json.zst
+        multi=gs://definitions.stackrox.io/v4/vulnerability-bundles/v1/vulnerabilities.zip
+        # Parse all supported pre-4.6 releases and copy the versioned bundle v1 to ensure
+        # these releases get updates.
+        grep -E "^[0-9]\.[0-9]+\.[0-9]+$" scanner/updater/version/RELEASE_VERSION | while read -r release; do
+          case "$release" in
+          4.4*)
```

Comment:
Nitpick:

```suggestion
          4.4.*)
```

It reads better, because otherwise it matches things like `4.41`. But, in practice will never happen so might be OK to leave as is.